client/packages/requisitions/src/ResponseRequisition/api/api.ts

```diff
@@ -105,6 +105,17 @@ const responseParser = {
     requestedQuantity: patch.requestedQuantity,
     optionId: patch?.reason?.id ?? null,
   }),
+  toListFilter: (
+    filterBy: FilterByWithBoolean | null
+  ): FilterByWithBoolean | null => {
+    Object.assign(filterBy ?? {},  {type: { equalTo: RequisitionNodeType.Response }});
+    if (filterBy?.['isEmergencyAndAutomatic']) {
+      filterBy['isEmergency'] = filterBy['isEmergencyAndAutomatic'];
+      filterBy['automaticallyCreated'] = filterBy['isEmergencyAndAutomatic'];
+    }
```

Comment:
I couldn't see if we have a pattern for multiple filters combined by one toggle already, so have mapped here.

I wanted to add is_emergency as one filter, but it needed to be split to two filters (is_emergency & automatically_created) in the back end.

Maybe there is a better pattern?

Have also moved the type filter up to the parser so it is all in one place.